torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -919,6 +925,275 @@ def get_b_layout(self):
         return LayoutType.VNNI2
 
 
+def woq_int4_extra_check(config, m, n, k, alpha, num_threads, **kwargs):
```

Comment:
`check_woq_int4_extra` seems align with the other function name.

Reply:
Thanks. Modified.